boards/f4vi1/include/periph_conf.h

```diff
@@ -78,28 +78,29 @@ static const timer_conf_t timer_config[] = {
 /** @} */
 
 /**
- * @name UART configuration
+ * @brief   UART configuration
```

Comment:
Actually, I have never spend much thought on this -> I have been using brief for everything for the last 2 years I think... 

About the spaces: they are there to have a unified indention in a doc comment, so if you use e.g. return, note, and brief in one block, their texts all start at the same col.